package.json

```
@@ -6,36 +6,32 @@
   "license": "Apache-2.0",
   "vendor": "Nuxeo",
   "devDependencies": {
+    "@open-wc/webpack": "^0.1.6",
     "@webcomponents/webcomponentsjs": "^2.0.0",
-    "del": "^2.1.0",
+    "clean-webpack-plugin": "^1.0.0",
+    "copy-webpack-plugin": "^4.6.0",
     "eslint-plugin-html": "^2.0.0",
-    "gulp": "^3.9.0",
-    "gulp-autoprefixer": "^3.1.0",
-    "gulp-cache": "^0.4.1",
-    "gulp-changed": "^1.3.0",
-    "gulp-debug": "^2.1.2",
-    "gulp-eslint": "^3.0.1",
-    "gulp-if": "^2.0.0",
-    "gulp-load-plugins": "^1.1.0",
-    "gulp-merge-json": "^0.6.0",
-    "gulp-rename": "^1.2.2",
-    "gulp-replace": "^0.5.4",
-    "gulp-size": "^2.0.0",
-    "gulp-useref": "^3.0.3",
-    "gulp-vulcanize": "^6.1.0",
-    "lodash": "^4.17.4",
-    "merge-stream": "^1.0.0",
-    "polymer-analyzer": "^2.7.0",
-    "polymer-build": "^3.1.0",
+    "npm-run-all": "^4.1.5",
     "polymer-cli": "^1.9.0",
-    "polyserve": "^0.27.11",
-    "run-sequence": "^1.1.5",
-    "through2": "^2.0.1",
-    "workbox-build": "^3.4.1"
+    "replace": "^1.0.1",
+    "webpack": "~4.28.4",
+    "webpack-bundle-analyzer": "^3.0.3",
+    "webpack-cli": "^3.1.2",
+    "webpack-dev-server": "^3.1.10",
+    "webpack-merge": "^4.1.4",
+    "workbox-cli": "3.6.3"
   },
   "scripts": {
-    "build": "gulp build",
-    "start": "gulp serve",
+    "start": "webpack-dev-server --env development",
```

If you run `start` without running build first, you won't get a  `.tmp` file, and we won't get merged labels either. We can fix it by adding a `prestart` script:
```suggestion
    "prestart": "mkdir -p .tmp && npm run prepare:i18n",
    "start": "webpack-dev-server --env development",
```